### ISSUE TITLE
No support currently for IE

### ISSUE THREAD

**Issue by arthur-1** [ombre-projects/ombre-website#6]
IE does not support CSS variables. Switch to a processor to fix this.

**Comment by yoosofan** [comment]
I do not think it is a big problem because most people who use cryptocurrency would use the latest version of new updated browsers because of security issues if they don't, then they should use the latest and new browsers for their sake.
